app/controllers/lettings_logs_controller.rb

```diff
@@ -80,13 +80,15 @@ def destroy
 
   def download_csv
     unpaginated_filtered_logs = filtered_logs(current_user.lettings_logs, search_term, @session_filters)
+    is_codes_only_export = params.require(:codes_only) == "true"
 
-    render "download_csv", locals: { search_term:, count: unpaginated_filtered_logs.size, post_path: email_csv_lettings_logs_path }
+    render "download_csv", locals: { search_term:, count: unpaginated_filtered_logs.size, post_path: email_csv_lettings_logs_path, is_codes_only_export: }
```

Comment:
write tests for this method in controller
 - [ ] parses is_codes_only_export correctly
 - [ ] displays page with eg button